addons/resource.language.en_gb/resources/strings.po

```diff
@@ -8646,8 +8661,10 @@ msgctxt "#19056"
 msgid "New timer"
 msgstr ""
 
+#. Label for a context menu entry to open timer settings dialog
+#: xbmc/pvr/windows/GUIWindowPVRTimers.cpp
 msgctxt "#19057"
-msgid "Edit timer"
+msgid "Edit settings"
```

Comment:
"New timer" DOES show up in context menu of a timer, as well as "Find similar" - I had checked that before writing ;) Question is if both make sense here. "Find similar" is ofc fine on EPG entries etc - just feels a bit odd on a timer. I think it's a leftover from before we had repeating timers in order to find the next airing of the show from a onetime timer. For this timer type it still might make sense - for rule based timers probably not (depends on the rules, if regex or not etc). We can ofc keep it as it might come in handy in times, it's just the labeling. "Find similar shows" might work on all places. We can discuss this in Slack to not spam everybody.